_includes/_shared_global_header.html

```diff
@@ -54,25 +54,26 @@
 </ul></li>
 <li class="col-sm-3 ">
 <ul><li class="dropdown-header">Media Categories</li>
-<li><a href="//mediaint.crossroads.net" data-automation-id="sh-allmedia">All Media</a></li>
-<li><a href="//mediaint.crossroads.net/series" data-automation-id="sh-series">Series</a></li>
-<li><a href="//mediaint.crossroads.net/music" data-automation-id="sh-music">Music</a></li>
-<li><a href="//mediaint.crossroads.net/videos" data-automation-id="sh-videos">Videos</a></li>
-<li><a href="//mediaint.crossroads.net/articles" data-automation-id="sh-articles">Articles</a></li>
-<li><a href="//mediaint.crossroads.net/podcasts" ui-sref="mediaint.podcasts" data-automation-id="sh-podcasts">Podcasts</a></li>
-<li><a href="//int.crossroads.net/live" target="_blank" data-automation-id="sh-livestreaming">Live Streaming</a></li>
+<li><a href="//media.crossroads.net" data-automation-id="sh-allmedia">All Media</a></li>
+<li><a href="//media.crossroads.net/series" data-automation-id="sh-series">Series</a></li>
+<li><a href="//media.crossroads.net/music" data-automation-id="sh-music">Music</a></li>
+<li><a href="//media.crossroads.net/videos" data-automation-id="sh-videos">Videos</a></li>
+<li><a href="//media.crossroads.net/articles" data-automation-id="sh-articles">Articles</a></li>
+<li><a href="//media.crossroads.net/podcasts" ui-sref="media.podcasts" data-automation-id="sh-podcasts">Podcasts</a></li>
+<li><a href="//www.crossroads.net/live" target="_blank" data-automation-id="sh-livestreaming">Live Streaming</a></li>
```

Comment:
Since this is going into int, could could you change these links back to mediaint.crossroads.net?